kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/producer/KafkaEventPublisher.java

```diff
@@ -93,8 +103,22 @@ public boolean supportsReset() {
      */
     public static class Builder<K, V> {
 
+        private String processingGroup;
         private KafkaPublisher<K, V> kafkaPublisher;
 
+        /**
+         * Sets the Kafka Event Handler processing group to be used by this {@link EventMessageHandler}
+         *
+         * @param processingGroup the Kafka Event Handler processing group to be used by this
+         *                        {@link EventMessageHandler}
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder<K, V> processingGroup(String processingGroup) {
+            assertNonNull(processingGroup, "ProcessingGroup may not be null");
```

Comment:
I would prefer `assertNonEmpty` here, to prevent using with an empty String.

Reply:
Good point! Changed accordingly!